tools/tools.go

```diff
@@ -162,33 +162,33 @@ func SockPostReqest(socketAddr string, API string, postValues string) ([]byte, e
 // SocketReqest makes a request to a unix socket
 func SocketReqest(socketAddr string, url string, method string, contentType string, body io.Reader) (*http.Response, error) {
 
-	log.Printf("[SOCK ] `%s` %s \"%s\"", socketAddr, method, url)
-	
+	log.Printf("[APP  ] Proxy `%s` %s \"%s\"", socketAddr, method, url)
```

Comment:
This function is a kind of generic function to call a usocket so I think the word APP proxy would be a bit confusing in future debug

Reply:
I inlined the 'SocketReqest' function into the 'HandleAppProxyRequest' from apps.go, so this function is not used atm.

Reply:
Ok good. I use this function to call docker socket as well